test/functional/cmdLineTests/modularityddrtests/modularityddrtests.xml

```diff
@@ -372,5 +372,182 @@
   <output regex="no" type="failure">DDRInteractiveCommandException</output>
 
  </test>
- 
- </suite>
\ No newline at end of file
+
+	<!-- Test !findmodules and all options -->
+	<test id="Run !findmodules">
+		<exec command="sh" capture="LOGNAME" platforms="zos.*" >
+			<arg>-c</arg>
+			<arg>echo $$LOGNAME</arg>
+		</exec>
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP.X001' $DUMPFILE$" platforms="zos_390-64.*" />
+		<exec command="cp //'$LOGNAME$.J9CORE.DMP' $DUMPFILE$" platforms="zos_390-31.*" />
+		<exec command="tso delete J9CORE.DMP.*" platforms="zos_390-64.*" />
+		<exec command="tso delete J9CORE.DMP" platforms="zos_390-31.*" />
+		<command command="$JDMPVIEW_EXE$">
+			<arg>-core $DUMPFILE$</arg>
```

Comment:
Perhaps the current framework doesn't care, but `-core` and `$DUMPFILE$` should be in separate `<arg>` elements. (Here and below).